algebra/tests/matrix/basic_interactions.c

```diff
@@ -44,24 +44,33 @@ static matrix_t dynMat;
 
 void algebraTests_checkInvalidSeek(matrix_t *M)
 {
-	int row, col;
+	int rowsNum, colsNum, row, col;
+
+	if (M->transposed) {
+		rowsNum = M->cols;
+		colsNum = M->rows;
+	}
+	else {
+		rowsNum = M->rows;
+		colsNum = M->cols;
+	}
 
 	/* Both row and col outside matrix */
-	TEST_ASSERT_NULL(matrix_at(M, M->rows, M->cols));
-	TEST_ASSERT_NULL(matrix_at(M, M->rows + SMALL_SHIFT, M->cols + SMALL_SHIFT));
-	TEST_ASSERT_NULL(matrix_at(M, M->rows + BIG_SHIFT, M->cols + BIG_SHIFT));
+	TEST_ASSERT_NULL(matrix_at(M, rowsNum, colsNum));
+	TEST_ASSERT_NULL(matrix_at(M, rowsNum + SMALL_SHIFT, colsNum + SMALL_SHIFT));
+	TEST_ASSERT_NULL(matrix_at(M, rowsNum + BIG_SHIFT, colsNum + BIG_SHIFT));
 
 	/* Only row outside matrix */
-	col = M->cols / 2;
-	TEST_ASSERT_NULL(matrix_at(M, M->rows, col));
-	TEST_ASSERT_NULL(matrix_at(M, M->rows + SMALL_SHIFT, col));
-	TEST_ASSERT_NULL(matrix_at(M, M->rows + BIG_SHIFT, col));
+	col = colsNum / 2;
+	TEST_ASSERT_NULL(matrix_at(M, rowsNum, col));
+	TEST_ASSERT_NULL(matrix_at(M, rowsNum + SMALL_SHIFT, col));
+	TEST_ASSERT_NULL(matrix_at(M, rowsNum + BIG_SHIFT, col));
 
 	/* Only col outside matrix */
-	row = M->rows / 2;
-	TEST_ASSERT_NULL(matrix_at(M, row, M->cols));
-	TEST_ASSERT_NULL(matrix_at(M, row, M->cols + SMALL_SHIFT));
-	TEST_ASSERT_NULL(matrix_at(M, row, M->cols + BIG_SHIFT));
+	row = rowsNum / 2;
```

Comment:
nitpick: add simple commentary something like:
```suggestion
	row = rowsNum / 2;  /* arbitrary position within rows */
```